perlin.py

```diff
@@ -77,16 +76,27 @@ def apply_network_noise(net: sumolib.net.Net, xml: ElementTree, scale: float, oc
     centre = find_city_centre(net)
     radius = radius_of_network(net, centre)
 
-    for edge in list(map(lambda x: x.getID(), net.getEdges())):
-        population = get_population_number(net=net, edge=edge, base=POPULATION_BASE, scale=scale, octaves=octaves,
-                                           centre=centre, radius=radius)
-        industry = get_population_number(net=net, edge=edge, base=INDUSTRY_BASE, scale=scale, octaves=octaves,
-                                         centre=centre, radius=radius)
-
-        for street in xml.find("streets").findall("street"):
-            if street.attrib["edge"] == edge:
-                street.set("population", str(population))
-                street.set("workPosition", str(industry))
+    streets = xml.find("streets")
+    if streets is None:
+        streets = ET.SubElement(xml.getroot(), "streets")
+
+    # Some edges might already have a street, so we want to ignore those
+    known_streets = {street.attrib["edge"]: street for street in streets.findall("street")}
+
+    for edge in net.getEdges():
+        eid = edge.getID()
+        if eid not in known_streets:
```

Comment:
So if an edge is not associated with a known street, we won't apply noisemap population and industry to it? 

I believe we should apply the noisemap to each street-element.

Reply:
But then we override anything that the user have put in the stats file manually.

Reply:
What we could do is adding an option for it. To override or not to override.

Reply:
Opened #10 as a response to this.